modular_skyrat/modules/importedvendors/code/vendors.dm

```diff
@@ -0,0 +1,182 @@
+/obj/effect/spawner/random/vending/snackvend
+	loot = list(
+		/obj/machinery/vending/imported,
+		///obj/machinery/vending/imported/panslav,
+		/obj/machinery/vending/imported/yangyu,
+		/obj/machinery/vending/imported/mothic,
+		/obj/machinery/vending/imported/tizirian,
+	)
+
+/obj/machinery/vending/imported
+	name = "NT Sustenance Supplier"
+	desc = "A vending machine serving up only the finest of human college student food."
+	icon = 'modular_skyrat/modules/importedvendors/icons/imported_vendors.dmi'
+	icon_state = "ntfood"
+	panel_type = "panel15"
+	light_mask = "ntfood-light-mask"
+	light_color = LIGHT_COLOR_LIGHT_CYAN
+	product_slogans = "Caution, contents may be selling hot!;Look at these low prices!;Hungry? Me too, wait no you didn't hear that!"
+	product_categories = list(
+		list(
+			"name" = "Snacks",
+			"icon" = "cookie",
+			"products" = list(
+				/obj/item/food/peanuts/random = 6,
+				/obj/item/food/cnds/random = 6,
+				/obj/item/food/pistachios = 6,
+				/obj/item/food/cornchips/random = 6,
+				/obj/item/food/sosjerky = 6,
+				/obj/item/reagent_containers/cup/soda_cans/cola = 6,
+				/obj/item/reagent_containers/cup/soda_cans/lemon_lime = 6,
+				/obj/item/reagent_containers/cup/soda_cans/starkist = 6,
+				/obj/item/reagent_containers/cup/soda_cans/pwr_game = 6,
+			),
+		),
+		list(
+			"name" = "Meals",
+			"icon" = "pizza-slice",
+			"products" = list(
+				/obj/item/storage/box/foodpack/nt = 6,
+				/obj/item/storage/box/foodpack/nt/burger = 6,
+				/obj/item/storage/box/foodpack/nt/chickensammy = 6,
+				/obj/item/food/vendor_tray_meal/side = 6,
+				/obj/item/food/vendor_tray_meal/side/crackers_and_jam = 6,
+				/obj/item/food/vendor_tray_meal/side/crackers_and_cheese = 6,
+			),
+		),
+	)
+
+	refill_canister = /obj/item/vending_refill/snack
+	default_price = PAYCHECK_CREW * 0.5
+	extra_price = PAYCHECK_COMMAND
+	payment_department = NO_FREEBIES
+
+	/// What language should this vendor speak, for flavor reasons
+	var/language_to_speak = /datum/language/common
+
+/obj/machinery/vending/imported/Initialize(mapload)
+	. = ..()
+	var/datum/language_holder/vendor_languages = get_language_holder()
+	grant_all_languages()
+	vendor_languages.selected_language = language_to_speak
+
+/* Pending me speaking to a russian (terrifying experience)
```

Comment:
Same note as above, might be best to remove the comments unless you know the PR will come in the near-future. On the off-chance someone comes in with their own addition or to clean the files, it could confuse them/get in the way

I need to break the habit too so I dont blame you for it lmao